### PR TITLE
[PROGRAMMES-6544] Add an offset class

### PR DESCRIPTION
**Pull request by benolot**
Unable to recreate this issue outside of the page to report to mozzila, but have fixed it in the page.

Ticket: https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6544

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Css/Core/Base/_main.scss [hunk 1]
```diff
@@ -56,7 +56,11 @@ abbr[title] {
 
 .programmes-page--inset,
 .inset {
-    padding-left: 8px;
+    padding-right: 8px;
```
Comment (votemike) [inline src/Ds2013/Css/Core/Base/_main.scss]: Why is this 8px, but if the page is in the bpw range, it's 0px?

File: src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/details.html.twig [hunk 2]
```diff
@@ -7,8 +7,13 @@
         {% else %}
             <h1 class="no-margin">{{ details.getEpisode().getTitle() }}</h1>
             <div class="gamma">
-                {{ ds_shared('entityContext', details.getEpisode(), {'include_self': false}) }}
-
+                {% if details.getEpisode().getParent().isTleo() %}
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/details.html.twig]: What was the reason for this `if`?

File: src/Ds2013/Css/Core/Base/_main.scss [hunk 3]
```diff
@@ -56,7 +56,11 @@ abbr[title] {
...
+}
+
+.offset {
```
Comment (votemike) [inline src/Ds2013/Css/Core/Base/_main.scss]: Is offset an appropriate name for this class? Is there a better and less generic name?

Comment (benolot) [reply src/Ds2013/Css/Core/Base/_main.scss]: Can't think of a better one, feel free to suggest

File: src/Ds2013/Css/Core/Base/_main.scss (hunk 1 above)
Comment (benolot) [reply src/Ds2013/Css/Core/Base/_main.scss]: Fixed it for the BPW range, it should still apply.

File: src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/details.html.twig (hunk 2 above)
Comment (benolot) [reply src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/details.html.twig]: Don't apply a right padding to the div if we don't want it to be padded (See the if on the spans where it sets inset)

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**